README.md

```diff
@@ -204,6 +204,14 @@ Note that by default, commit messages need to conform to a strict pattern specif
 
 Note that this regex requires a period at the end of the commit message.
 
+## Installing, deploying, and enabling a new module
+
+1. Install and enable the module locally
+2. Export configuration locally (eg, `drush @dojportal-blt.local cex`)
+3. Commit the exported configuration
+4. Deploy to Acquia with `blt deploy`
```

Comment:
We were using the command `blt artifact:deploy --commit-msg "BLT-00: commit message." --branch "master" --no-interaction`

Reply:
@LindsayYoung Ooooh ... good catch.

Reply:
@LindsayYoung Updated.